### PR TITLE
Close #117. UI Refactoring of Timer.

### LINKED ISSUE #117: UI Refactoring of Timer 🎯 2 days

### PR DESCRIPTION
**Pull request by kaxcode**
Implement CSS Grid into AdjustTimerButton, ControllerButtons, and Timer.
Use font-awesome with CDN in index.html file.
Add google font into project via index.html file.
Remove react-materialize after CSS grid implementation.
Add Timer Count Colon between Hours- Minutes, MInutes-Seconds.
Add extra margin and enlarge cloud logo.
Overall CSS updates in Timer.
Update snapshot tests.

### CODE REVIEW

File: src/components/Timer/__tests__/__snapshots__/Timer.test.js.snap [hunk 1]
```diff
@@ -30,11 +28,21 @@ exports[`Timer renders correctly 1`] = `
       >
         0
       </div>
+      <div
+        className="Timer__Colon"
```
[the remark this answers is not in the document]
Comment (kaxcode) [reply src/components/Timer/__tests__/__snapshots__/Timer.test.js.snap]: Changes to Timer__Delimiter

File: src/components/ControllerButtons/ControllerButtons.js [hunk 2]
```diff
@@ -1,37 +1,36 @@
 import React from 'react';
 import './ControllerButtons.css';
-import { Button } from 'react-materialize';
 import { func, bool } from 'prop-types';
 
 const ControllerButtons = props => {
   return (
     <div className="ControllerButtons">
-      <Button
+      <div
```
Comment (dlindahl) [inline src/components/ControllerButtons/ControllerButtons.js]: No. No no no. @meingorn, tell Kenia why this is a no.

Comment (kaxcode) [reply src/components/ControllerButtons/ControllerButtons.js]: @dlindahl is it supposed to be a <button> and not <div>?

Comment (meingorn) [reply src/components/ControllerButtons/ControllerButtons.js]: accessibility! div's aren't strictly-speaking clickable - this is a clickable element and thus should be a <button />

**Pull request merged**